### ISSUE TITLE
Trailing spaces in inferred operationName on GraphQL query causing issues

### ISSUE THREAD

**Issue by peteoshea** [Huachao/vscode-restclient#960]
- REST Client Version: v0.24.6
- VSCode Version: 1.63.2
- OS Version: Windows 11

Steps to Reproduce:

1. Create a GraphQL query e.g.:
```gql
POST {{$dotenv GRAPHQL_URL}}
Content-Type: application/json
Authorization: Bearer {{token}}
X-REQUEST-TYPE: GraphQL

query products {
  products {
    id
    name
  }
}
```
2. When running this query an operationName is inferred including a trailing space which seems to break the GraphQL call:
```json
{
  "query": "query products {\n  products {\n    id\n    name\n  }\n}",
  "operationName": "products ",
  "variables": {}
}
```
expected:
```json
{
  "query": "query products {\n  products {\n    id\n    name\n  }\n}",
  "operationName": "products",
  "variables": {}
}
```

If the space is removed between the operation name and the curly brace, i.e. `query products{`, then the GraphQL query works fine. The space seems to be the recommended way to format GraphQL queries though and as this has worked fine for us in the past I consider this a bug. I guess this was introduced as part of the work for #919

**Comment by Huachao** [comment]
@peteoshea nice catch, this fix will be published in the next release

**Comment by Huachao** [comment]
@peteoshea this has been fixed in the latest version 0.25.0
